GPU/GPUTracking/Merger/GPUTPCGMO2Output.cxx

```diff
@@ -134,6 +136,10 @@ GPUdii() void GPUTPCGMO2Output::Thread<GPUTPCGMO2Output::output>(int nBlocks, in
     if (merger.Param().par.dodEdx) {
       oTrack.setdEdx(tracksdEdx[i]);
     }
+
+    auto pid = pidResponse.getMostProbablePID(oTrack);
+    oTrack.setPID(pid);
+
     oTrack.setOuterParam(o2::track::TrackParCov(
       outerPar.X, outerPar.alpha,
       {outerPar.P[0], outerPar.P[1], outerPar.P[2], outerPar.P[3], outerPar.P[4]},
```

Comment:
The setting of the PID should be done after `setOuterParam` and should set the PID also for outer params:
```suggestion

    oTrack.setOuterParam(o2::track::TrackParCov(
      outerPar.X, outerPar.alpha,
      {outerPar.P[0], outerPar.P[1], outerPar.P[2], outerPar.P[3], outerPar.P[4]},
      
    const auto pid = pidResponse.getMostProbablePID(oTrack);
    oTrack.setPID(pid);
    oTrack.getOuterParam().setPID(pid);
```